src/css/support.js

```diff
@@ -46,61 +46,57 @@ define([
 		documentElement.removeChild( container );
 	}
 
-	// Support: node.js jsdom
-	// Don't assume that getComputedStyle is a property of the global object
```

Comment:
Yup, `window` is not a global here but what was passed to the factory. Unless the global has a property `document` when we assume we're in a browser and assign `window = current global`.

This works without this `if` in jsdom because its `window` parameter has `getComputedStyle`. Do we want to support anything else here?